src/datasources/cache/redis.cache.service.spec.ts

```diff
@@ -173,4 +173,20 @@ describe('RedisCacheService', () => {
     expect(ttl).toBeGreaterThan(0);
     expect(ttl).toBeLessThanOrEqual(expireTime);
   });
+
+  it('increments the value of an existing key', async () => {
```

Comment:
This is a duplicate of the test above.

Reply:
Oops, it got duplicated in a rebase. Thanks!

Removed in [23d9d5e](https://github.com/safe-global/safe-client-gateway/pull/1228/commits/23d9d5e6730da71544fe9b883c2cc35d8fe55fa1)